carp.data.core/src/commonTest/kotlin/dk/cachet/carp/data/infrastructure/StudyDataServiceRequestTest.kt

```diff
@@ -0,0 +1,59 @@
+package dk.cachet.carp.data.infrastructure
+
+import dk.cachet.carp.common.application.UUID
+import dk.cachet.carp.data.application.CollectedDataQuery
+import kotlinx.datetime.Instant
+import kotlinx.serialization.json.Json
+import kotlin.test.*
+
+class StudyDataServiceRequestTest
```

Comment:
I don't remember how exactly I tested the other services, but look at those to get a feel for the right level of coverage and test setups. 

Align them/be consistent, unless there is a specific reason to do something different. 

I seem to recall adding more generic test setups through the use of base classes to more quickly provide coverage.